src/transformers/modeling_utils.py

```diff
@@ -666,6 +677,64 @@ def _get_resized_embeddings(
 
         return new_embeddings
 
+    def _get_resized_lm_head(
```

Comment:
Not super fan of the name as it's the result of `get_output_embeddings`. `_get_resized_decoder` sounds better to me (but it's a pure nit so feel free to ignore me).